src/MainWindow.vala

```diff
@@ -132,7 +137,12 @@ public class Camera.MainWindow : Gtk.ApplicationWindow {
 
         camera_content.set_player (camera_view);
 
-        loading_view.set_status (_("Connecting to \"%s\"…").printf (camera_view.get_camera_device ().get_name ()));
+         if  (camera_view.get_camera_device ().get_name () == null) {
+            loading_view.set_status (_("Connecting to %s…").printf ("camera"));
+        } else {
+            loading_view.set_status (_("Connecting to \"%s\"…").printf (camera_view.get_camera_device ().get_name ()));
```

Comment:
I think it's fine to lose the quotes around this camera name. I dropped them from the previous line because it felt awkward to quote the word “camera” in that context but I think it works either way.